client/src/services/entityService.ts

```diff
@@ -0,0 +1,36 @@
+import type { Entity } from '../../types/entityType';
+import type { Service } from '../../types/ServiceType';
+
+export default class EntityService {
+    static async getEntities(type = ''): Promise<Entity[]> {
+        const headers = new Headers();
+        headers.set('Content-Type', 'application/json');
+
+        if (!type) {
+            const response = await fetch('http://localhost:3000/entity', {
```

Comment:
I believe the URL to call is declared in a config file

Reply:
Done